internal/factsengine/gatherers/ini_files_test.go

```diff
@@ -0,0 +1,287 @@
+package gatherers_test
+
+import (
+	"context"
+	"testing"
+
+	st "github.com/balanza/supertouch"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/suite"
+	"github.com/trento-project/agent/internal/factsengine/gatherers"
+	"github.com/trento-project/agent/pkg/factsengine/entities"
+)
+
+type IniFilesTestSuite struct {
+	suite.Suite
+}
+
+func TestIniFilesTestSuite(t *testing.T) {
+	suite.Run(t, new(IniFilesTestSuite))
+}
+
+func (suite *IniFilesTestSuite) TestIniFilesGathererEmptyArgumentProvided() {
+	c := gatherers.NewDefaultIniFilesGatherer()
+
+	factRequests := []entities.FactRequest{
+		{
+			Name:     "any",
+			Gatherer: "ini_files",
+		},
+	}
+
+	_, err := c.Gather(context.Background(), factRequests)
+
+	suite.Error(err)
```

Comment:
nitpick: I think we can improve the error checking with `suite.EqualError`. This is for all the tests returning errors